pkg/api/handlers/compat/containers.go

```diff
@@ -403,22 +403,24 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		state.Status = define.ContainerStateCreated.String()
 	}
 
-	state.Health = &types.Health{
-		Status:        inspect.State.Healthcheck.Status,
-		FailingStreak: inspect.State.Healthcheck.FailingStreak,
-	}
-
-	log := inspect.State.Healthcheck.Log
+	if l.HasHealthCheck() && state.Status != "created" {
```

Comment:
`define.ContainerStateConfigured.String()` instead of "created"

Reply:
Existing code is probably safer, otherwise we potentially conflate ContainerStateConfigured and ContainerStateCreated